packages/@react-spectrum/contextualhelp/stories/ContextualHelp.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {action} from '@storybook/addon-actions';
+import {Button, Content, Flex, Footer, Heading, Link, Text} from '@adobe/react-spectrum';
+import {ContextualHelp} from '../src';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+storiesOf('ContextualHelp', module)
+.add(
+  'default',
+  () => render({heading: 'Help title', description: helpText()})
+)
+.add(
+  'type: info',
+  () => render({heading: 'Help title', description: helpText(), variant: 'info'})
+)
+.add(
+  'with link',
+  () => render({heading: 'Help title', description: helpText(), link: <Link>Learn more</Link>})
+)
+.add(
+  'with button',
+  () => (<Flex alignItems="center">
+    <Button variant="primary" isDisabled>Create</Button>
+    <ContextualHelp>
+      <Heading>Help title</Heading>
```

Comment:
Looks like this changed the API slightly from what was in #2352, where the title was a prop and the content was the only children. What was the reason for changing that?

Reply:
When we left it up to the user to include any links they wanted as part of the content that meant they were also responsible for the spacing between the content text and the link. By using the footer slot we were able to better control the spacing between them. Then since the content and footer slots were now being used having title as a prop looked odd so it made sense to just use all three existing dialogs slots for `ContextualHelp` as well.

The downside here is that we have exposed the full dialog implementation which makes `ContextualHelp` a little more complex to use vs props.

Reply:
We decided this change was ok. It may be a little bit more complex, but I think if people are familiar with our dialogs it should be ok.